cardano-cli/src/Cardano/CLI/Run.hs

```diff
@@ -48,16 +49,20 @@ data ClientCommand =
     -- now-deprecated \"shelley\" subcommand.
   | DeprecatedShelleySubcommand ShelleyCommand
 
+  | CliPingCommand PingCmd
+
   | forall a. Help ParserPrefs (ParserInfo a)
   | DisplayVersion
 
 data ClientCommandErrors
   = ByronClientError ByronClientCmdError
   | ShelleyClientError ShelleyCommand ShelleyClientCmdError
+  | PingError PingCmdError
```

Comment:
Why are you breaking away from the current pattern?

Reply:
Renamed to `PingClientError`.